Upstream fixes already included in the latest community updates to coolkey v1.1.0

Adds additional headers that are needed for log fixes.

--- ORIGINAL/./src/coolkey/log.cpp	2016-06-24 16:07:19.418986402 -0400
+++ ././src/coolkey/log.cpp	2016-06-24 11:46:48.985184817 -0400
@@ -18,6 +18,8 @@
  * ***** END COPYRIGHT BLOCK *****/
 
 #include <string>
+#include <string.h>
+#include <stdlib.h>
 #include "mypkcs11.h"
 #include <assert.h>
 #include <stdio.h>
